Misc/NEWS.d/next/Security/2021-01-31-05-28-14.bpo-43075.DoAXqO.rst

```diff
@@ -0,0 +1 @@
+Fix ReDoS in request
```

Comment:
Please be more informative. If you are Python user reading changelog, what it would say to you? What request? What type of ReDoS? Does it affect your code if it does so and so, or you can ignore it?